src/fides/api/models/privacy_notice.py

```diff
@@ -153,15 +154,6 @@ class PrivacyNotice(PrivacyNoticeBase, Base):
         order_by="NoticeTranslation.created_at",
     )
 
-    # Forward declaration to avoid circular import errors
-    PrivacyExperienceConfig = "PrivacyExperienceConfig"
-
-    experience_configs = relationship(
-        PrivacyExperienceConfig,
-        secondary="experiencenotices",
-        back_populates="privacy_notices",
-    )
```

Comment:
Changing `back_populates` to `backref` on `PrivacyExperienceConfig.privacy_notices` creates this relationship automatically on the `PrivacyNotice` right?

Reply:
yes! `backref` is no longer the [preferred way of doing this](https://docs.sqlalchemy.org/en/20/orm/relationship_api.html#sqlalchemy.orm.relationship.params.backref) I think you're supposed to use back_populates now because it's more straightforward, but I think this is a viable option for now for these two tightly linked models